src/components/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import style from './ToDoList.module.css'
+
+interface PropsType {
+  todo: {
+    id: number
+    task: string
+    complete: boolean
+  }
+  removeTask: (newId: number) => void
+  completeTask: (newId: number) => void
+}
+
+const ToDoList: React.FC<PropsType> = (props) => {
+  return (
+    <div key={props.todo.id} className={style.todoList}>
+      <div
+        title={props.todo.complete ? 'click if not complete' : 'click if complete'}
+        className={props.todo.complete ? `${style.task} ${style.complete}` : style.task}
```

Comment:
try this

        className={`${style.task} ${props.todo.complete ? style.complete : ''}`}